xbmc/settings/dialogs/GUIDialogSettingsManualBase.cpp

```diff
@@ -239,6 +245,24 @@ CSettingAction* CGUIDialogSettingsManualBase::AddButton(CSettingGroup *group, co
   return setting;
 }
 
+CSettingString* CGUIDialogSettingsManualBase::AddInfoStringButton(CSettingGroup *group, const std::string &id, int label, int level, std::string info, bool delayed /* = false */,
```

Comment:
The idea of `delayed` is that it takes a bit before the changed value of a setting takes effect. This makes sense for edit controls where the user can type or for spinners where the user might want to spin through all the available options without immediately seeing the effect. I don't see how that makes any sense for an  info label which can't be changed by the user.